app/templates/search/_services_results.html

```diff
@@ -9,7 +9,7 @@ <h2 class="search-result-title">
     </p>
 
     <p class="search-result-excerpt">
-        {{ service.serviceSummary or service.serviceDescription }} {# G-Cloud 9 uses serviceDescription, not serviceSummary. #}
```

Comment:
I know that in theory only G9 services will come back from search, but feels like one that we could leave in as a "just in case"?